packages/manager/apps/pci-instances/src/components/cart/components/CartContent.component.tsx

```diff
@@ -0,0 +1,39 @@
+import { Accordion, AccordionChangeDetail } from '@ovhcloud/ods-react';
+import { ReactNode, useCallback, useState } from 'react';
+import { TCartItem } from '../Cart.component';
+
+export type TRenderItemParams = {
+  item: TCartItem;
+  isExpanded: (accordionName: string) => boolean;
+};
+
+export type TCartContentProps = {
+  items: TCartItem[];
+  renderCartItem: (params: TRenderItemParams) => ReactNode;
+};
+
+export const CartContent = ({ items, renderCartItem }: TCartContentProps) => {
+  const [expandedAccordions, setExpandedAccordions] = useState<string[]>(
+    items.filter((item) => item.expanded).map((item) => item.title),
+  );
+
+  const handleAccordionChange = useCallback(
+    (accordion: AccordionChangeDetail) =>
+      setExpandedAccordions(accordion.value),
+    [setExpandedAccordions],
+  );
+
+  const isExpanded = useCallback(
+    (accordionName: string) => expandedAccordions.includes(accordionName),
+    [expandedAccordions],
+  );
```

Comment:
IMO there is no need to memoize with `useCallback`

Reply:
clearly not!